src/@services/AVSService.js

```diff
@@ -37,4 +37,15 @@ export default class AVSService {
     // TODO: Handle error
     return await response.json();
   }
+
+  async getAvsTvlOvertime(address) {
```

Comment:
there is this comment [here](https://github.com/NethermindEth/restaking-dashboard/pull/93#discussion_r1655157835) about following naming conventions, but it would be ridiculous to name this `getAVSTVLOvertime`. Ruben suggested to use `getAVSTotalValue` and drop the "Overtime" 
```suggestion
  async getAVSTotalValue(address) {
```